archive/test-archives/DATABASE_MIGRATION_PLAN.md

```
@@ -0,0 +1,187 @@
+            applicant_data=json.dumps(application_data["applicant_data"]),
+            status=application_data["status"],
+            applied_at=application_data["applied_at"]
+        )
+        self.db.add(db_application)
+        self.db.commit()
+        return db_application
+    
+    def get_applications_by_property(self, property_id: str) -> List[JobApplication]:
+        return self.db.query(JobApplication).filter(
+            JobApplication.property_id == property_id
+        ).all()
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)